app/pom.xml

```diff
@@ -55,7 +55,6 @@ limitations under the License.
         <maven-antrun.version>1.0b3</maven-antrun.version>
         <rome.version>1.13.1</rome.version>
         <slf4j.version>1.7.30</slf4j.version>
-        <spring.version>5.2.7.RELEASE</spring.version>
```

Comment:
was this intended? spring and spring sec are now defined in two different poms. (asking before i rebase)

does this help to reduce selenium warnings or has it other reasons?

Reply:
I was trying to use dependencyManagement in the parent POM to force all Spring deps to the same version, but I don't think it really helped with all those warnings you see when Jetty starts in app or it-selenium. 

I'm going to back this change out.